packages/plugin-lighthouse/src/lib/runner/utils.unit.test.ts

```diff
@@ -126,14 +128,45 @@ describe('toAuditOutputs', () => {
     );
   });
 
+  it('should not parse given audit details', () => {
+    expect(
+      toAuditOutputs(
+        [
+          {
+            id: 'cumulative-layout-shift',
+            details: {
+              type: 'table',
+              headings: [{ key: 'number' }],
+              items: [{ number: 42 }],
+            },
+            score: 0,
+            numericValue: 0,
+            displayValue: '0 ms',
+          } as unknown as Result,
+        ],
+        { verbose: true },
+      ).at(0)?.details,
+    ).toStrictEqual({
+      table: {
+        columns: [{ key: 'number', align: 'left' }],
+        rows: [{ number: 42 }],
+        title: 'Table',
+      },
+    });
+  });
+
   it('should NOT inform for unsupported details if verbose is NOT given', () => {
     const types = [...unsupportedDetailTypes];
     toAuditOutputs(
       types.map(
         type =>
           ({
-            details: { type },
-          } as Result),
+            id: 'cumulative-layout-shift',
+            details: { type, headings: [], items: [] },
+            score: 0,
+            numericValue: 0,
+            displayValue: '0 ms',
+          } as unknown as Result),
```

Comment:
There's always a better way than `as unknown as T` (part 2):

```suggestion
            id: 'cumulative-layout-shift',
            details: { type, headings: [], items: [] } as Details,
            score: 0,
            numericValue: 0,
            displayValue: '0 ms',
          } as Result),
```